sherpa/cpp_api/brpc/client.cc

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright      2022  (authors: Pingfeng Luo)
+ *
+ * See LICENSE for clarification regarding multiple authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <stdio.h>
+
+#include "asr.pb.h" //NOLINT
+#include "brpc/channel.h"
+#include "brpc/stream.h"
+#include "bthread/bthread.h"
+#include "butil/logging.h"
+#include "gflags/gflags.h"
+
+
+DEFINE_string(protocol, "baidu_std",
```

Comment:
Can we remove gflags and use `parse_options.{h,cc}` to replace it?

Reply:
ok, done.